src/main/java/org/apache/hadoop/io/MaxTextLineLengthExceededIOException.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.hadoop.io;
+
+import java.io.IOException;
+
+public class MaxTextLineLengthExceededIOException
```

Comment:
Let's drop the "IO" in here, since that sounds strange. How about `TextLineLengthLimitExceededException`